Luke Smith Portfolio.css

```diff
@@ -0,0 +1,111 @@
+/** {
+	box-sizing: border-box;
+}
+*/
+body {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: black;
+  margin: 0;
+  max-width: 500px;
+  margin: auto;
+  padding: 10px;
+}
+
+.column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  margin: auto;
+}
```

Comment:
I also can't find any elements in your html with a class of "card", so this styling rule is also not doing anything. Is it a remnant of a previous design concept? It can be safely deleted now.